content/blog/newsletter/2025-09-05-newsletter.md

```diff
@@ -0,0 +1,189 @@
+- [Jabbers.one](https://jabbers.one) The server has been migrated to the hosting provider Netcup. A public monitoring status page is now available at: [https://xmpp-status.fluxxer.de](https://xmpp-status.fluxxer.de). Also a Grafana statistic site is now available at [https://stats.jabbers.one](https://stats.jabbers.one).
+- [jabjab.de](https://www.jabjab.de/) XMPP server celebrates [20 years of existence](https://www.jabjab.de/blog#96)! [DE]
+- [MongooseIM 6.4: Simplified and Unified](/2025/08/mongooseim-6.4-simplified-and-unified/) by [Pawel Chrzaszcz](https://www.erlang-solutions.com/author/pawel-chrzaszcz/) for [Erlang Solutions](https://www.erlang-solutions.com/).
+- [O protocolo mais perfeito dos imperfeitos](https://blog.isacloud.im/o-protocolo-mais-perfeito-dos-imperfeitos/) by [isadora](https://blog.isacloud.im/author/isadora/). An overview of the benefits of XMPP, including tips on how to introduce it to new people who may not be interested in the technical details. [PT_BR]
+- [Cambios en mi red de la mano de XMPP](https://gvisoc.com/posts/cambios-xmpp/) by [Gabriel Viso Carrera](https://fedi.gvisoc.com/@gabriel) for [gvisoc.com](https://gvisoc.com/acerca-de/). [[EN](https://gvisoc.com/en/posts/changes-xmpp/)]
+- Wondering what [Monocles](https://monocles.eu/) has in store for their next version? Besides of the array of assorted enhancements and fixes that Monocles includes in every new release, the upcoming version will include support for [XEP-0118](/extensions/xep-0118.html) (User Tune), enabling its users to optionally share with their contacts the music they are currently listening to. This feature enhances social interaction by allowing users to discover new music, share their tastes, and connect over shared interests, fostering a more engaging and collaborative experience within the app. Check out the [original announcement](https://monocles.social/@monocles/115068019551027353)! A big thank you to the developer who contributed to this feature!
+
+## XMPP Software News
+
+### XMPP Clients and Applications
+
+- [Cheogram](https://cheogram.com/) has released version [2.19.0-1](https://git.sr.ht/~singpolyma/cheogram-android/refs/2.19.0-1) for Android. Make sure to check out the [changelog](https://git.sr.ht/~singpolyma/cheogram-android/tree/master/item/CHANGELOG.md) for all the details.
+- [Conversations](https://conversations.im/) has released versions [2.19.3](https://codeberg.org/iNPUTmice/Conversations/releases/tag/2.19.3) and [2.19.4](https://codeberg.org/iNPUTmice/Conversations/releases/tag/2.19.4) for Android. This version brings improved performance for large public channels, a new 'empty chats' page instead of redirect to 'new chat' screen, and **requires TLS 1.3 for all connections with setting to opt out**, among other fixes and improvements. You can take a look at the [changelog](https://codeberg.org/iNPUTmice/Conversations/src/branch/master/CHANGELOG.md) for all the details.
```

Comment:
```suggestion
- [Conversations](https://conversations.im/) has released versions [2.19.3](https://codeberg.org/iNPUTmice/Conversations/releases/tag/2.19.3) and [2.19.4](https://codeberg.org/iNPUTmice/Conversations/releases/tag/2.19.4) for Android. This version brings improved performance for large public channels, a new 'empty chats' page instead of redirect to 'new chat' screen, and **requires TLS 1.3 for all connections with a setting to opt out**, among other fixes and improvements. You can take a look at the [changelog](https://codeberg.org/iNPUTmice/Conversations/src/branch/master/CHANGELOG.md) for all the details.
```